### PR TITLE
docs(ecosystem): add fastify-aws-timestream and fastify-aws-sns

### PR DESCRIPTION
**Pull request by gzileni**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by mcollina** [comment]
the linter is failing

**Comment by gzileni** [comment]
I fixed all the bugs and made another PULL request

**Comment by mcollina** [comment]
the linter is still failing

**Comment by Uzlopak** [comment]
fixed linting issues

File: docs/Guides/Ecosystem.md [hunk 1]
```diff
@@ -218,6 +218,12 @@ section.
   directory.
 - [`fastify-axios`](https://github.com/davidedantonio/fastify-axios) Plugin to
   send HTTP requests via [axios](https://github.com/axios/axios).
+- [fastify-aws-timestream](https://github.com/gzileni/fastify-aws-timestream)
+  Fastify plugins to managing databases, tables and querying and creating
+  scheduled queries with AWS Timestream
+- [fastify-aws-sns](https://github.com/gzileni/fastify-aws-sns) Fastify plugin
+  to AWS Simple Notification Service (AWS SNS) to coordinates and manages
+  the delivery or sending of messages to subscribing endpoints or clients
```
Comment (Fdawgs) [inline docs/Guides/Ecosystem.md:226]:
```suggestion
- [fastify-aws-timestream](https://github.com/gzileni/fastify-aws-timestream)
  Fastify plugin for managing databases, tables, and querying and creating
  scheduled queries with AWS Timestream.
- [fastify-aws-sns](https://github.com/gzileni/fastify-aws-sns) Fastify plugin
  for AWS Simple Notification Service (AWS SNS) that coordinates and manages
  the delivery or sending of messages to subscribing endpoints or clients.
```

**Review by Fdawgs — CHANGES_REQUESTED** [review]

File: docs/Guides/Ecosystem.md (hunk 1 above)
Comment (Uzlopak) [reply docs/Guides/Ecosystem.md:226]: Is that the bloody famous oxford comma, Sir Frazer Smith?
image

Comment (Fdawgs) [reply docs/Guides/Ecosystem.md:226]: > Is that the bloody famous oxford comma, Sir Frazer Smith? image

Yeah boi 😄

Comment (gzileni) [reply docs/Guides/Ecosystem.md]: I fixed all the bugs. Thanx

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**